src/events/sampleEvent.ts

```diff
@@ -1,5 +1,6 @@
-// set the function name to the event name 
+// set the function name to the event name
 // and edit the parameter to fit in its type and name to the event
 export default async function interactionName(args: unknown) {
-    // do things if the event occurs 
+	// do things if the event occurs
+	console.log(args);
```

Comment:
and don't forget to import the logger
```suggestion
	logger.debug(args);
```